### DISCUSSION TITLE
Random alias should be able to randomly choose a domain

### DISCUSSION THREAD

**Discussion by proletarius101** [simple-login/app#433] [Feature Requests]
When generating a random alias, simple login always sticks to one domain, which leaks a relatively strong fingerprint. It should be able to randomly choose a domain in the pool.

**Comment by nguyenkims** [comment]
You can choose the domain that's used generate new random aliases on the Settings page at https://app.simplelogin.io/dashboard/setting

![image](https://user-images.githubusercontent.com/3167780/114672468-78ae0580-9d05-11eb-984e-dcb0e01be9cb.png)

**Reply by proletarius101** [reply]
Well, that's the context of this problem. You can choose a different default domain (which is great to reduce the chance to be blocked), but you will use that domain to generate email addresses unless you change it. If only a small amount of people use the domain you choose (e.g. you live in a country where very few people use simple login), you're easy to track.
And the proposal is, we should be able to randomly choose a domain, which could increase the entropy.

**Reply by nguyenkims** [reply]
Oh I see, thanks for the explanation. Do you think having an option "Randomize" in the below screenshot that picks a domain randomly when creating a random alias would be useful? 

![2021-04-14 at 12 13](https://user-images.githubusercontent.com/3167780/114694421-cbde8300-9d1a-11eb-9c24-3616a41f4b02.png)

**Reply by proletarius101** [reply]
That's definitely great to have :+1:
